app/models/physical_rack.rb

```diff
@@ -3,9 +3,9 @@ class PhysicalRack < ApplicationRecord
 
   belongs_to :ext_management_system, :foreign_key => :ems_id, :inverse_of => :physical_racks,
     :class_name => "ManageIQ::Providers::PhysicalInfraManager"
-
   has_many :physical_chassis, :dependent => :nullify, :inverse_of => :physical_rack
   has_many :physical_servers, :dependent => :nullify, :inverse_of => :physical_rack
+  has_many :physical_storages, :dependent => :nullify, :inverse_of => :physical_rack
```

Comment:
This is just adding an association that already existing on the physical_storage model right?  Nothing was changed in schema for this to work.

Reply:
Yes, it doesn't change nothing in schema. This was done because `PhysicalRack` requires an association with `PhysicalStorage` in order to get them when needed. If this is not added, then we can get a `PhysicalRack` from a `PhysicalStorage`, but the inverse does not work.